1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,58 @@
+// function readingList(books) {
+//   let content = document.querySelector("#content");
+//   const listBooks = [];
+
+//   for (let book of books) {
+//     console.log(book);
+//     let ul = document.createElement("ul");
+//     let li = document.createElement("li");
+//     listBooks.push(ul);
+//     ul.innerHTML = `
+//       <li class= list>
+//       <p>${book.title} by ${book.author}</p>
+//       <img class=image src= "${book.bookCoverImage}" >
+//       </li>
+//     `;
+//     content.appendChild(ul);
+//   }
+// }
+
 function readingList(books) {
-  // Write your code here...
+  let ul = document.createElement("ul");
+  ul.setAttribute("id", "proList");
+  let content = document.querySelector("#content");
+
+  for (book of books) {
+    let li = document.createElement("li");
+    li.setAttribute("class", "item");
+    ul.appendChild(li);
+    li.innerHTML = `
+    <p>${book.title} by ${book.author}</p>
+          <img class=image src= "${book.bookCoverImage}" >`;
+  }
+  content.appendChild(ul);
 }
 
```

Comment:
Nice work Marina!  I would just delete any commented out code so it looks cleaner and helps find the relevant code faster and easier :)